src/main/java/seedu/address/model/entity/body/DonationList.java

```diff
@@ -0,0 +1,9 @@
+package seedu.address.model.entity.body;
+
+//@@author ambervoong
+/**
+ * Enumerates body parts that can be donated.
+ */
+public enum DonationList {
+    KIDNEYS, HEART, LUNGS, LIVER, SMALLBOWEL, PANCREAS, EYES, HEARTVALVES, BONE, SKIN, VEINS, TENDONS
```

Comment:
According to this source: https://www.transplantatiestichting.nl/english/10-facts-about-organ-and-tissue-donation , parts like the corneas, skin, veins, heart valves, tendons, ligaments and bones are considered tissues. Should we separate organ and tissue? Or maybe collectively call them bodypart?  Also I think we should standardise using singular/plural for the body parts

Reply:
Maybe we can do a collective class like this one and rename it to bodypart? Yes, I'll change that. But do you think we should even bother with enums for this, considering that most of the enums (for cause of death, relationship) have been removed because there were too many choices?

Reply:
Changed to ArrayList<String>.